mautic/mautic
# Fix form field campaign condition with special character in value

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Form field condition made wrong condition If value contain special character like `&`



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Crete campaign form with email field
3. Create select field with one value `test & test`
4. Then create campaign with source of this form
5. Then create condition form field value with you select field condition equal `test & test`
6. Submit form
7. Trigger campaign actions
7. See your contact was not passed by condition
8. After PR should go to true
